boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,168 @@
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/lpcxpresso54114/lpcxpresso54114_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC54114 SoC has IOCON registers, which can be used to configure the
+functionality of a pin.
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_0  | USART           | USART RX                   |
++---------+-----------------+----------------------------+
+| PIO0_1  | USART           | USART TX                   |
++---------+-----------------+----------------------------+
+| PIO0_29 | GPIO            | RED LED                    |
++---------+-----------------+----------------------------+
+| PIO1_9  | GPIO            | BLUE_LED                   |
++---------+-----------------+----------------------------+
+| PIO1_10 | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The LPC54114 SoC is configured to use the internal FRO at 48MHz as a source for
+the system clock. Other sources for the system clock are provided in the SOC,
+depending on your system requirements.
+
+Serial Port
+===========
+
+The LPC54114 SoC has 8 FLEXCOMM interfaces for serial communication.  One is
+configured as USART for the console and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+The LPCXpresso54114 includes the LPC-Link2 serial and debug adapter built into
+the board to provide debugging, flash programming, and serial communication
+over USB. LPC-Link2 can be configured with Segger J-Link or CMSIS-DAP firmware
+variants to support corresponding debug tools. Currently only the Segger J-Link
+tools are supported for this board in Zephyr, therefore you should use the
+Segger J-Link firmware variant.
+
+Before you start using Zephyr on the LPCXpresso54114, download and run
+`LPCScrypt`_ to update the LPC-Link2 firmware to the latest version. Serial
+communication problems, such as dropping characters, have been observed with
+older versions of the firmware.
```

Comment:
can we spec a version here about the firmware.  I'm wondering when someone reads this a year from now or two, will the comment still be valid?  Seems like if we can reference some versions it would be helpful.  (ie needing firmware version > X.Y.Z)

Reply:
Problem is that I don't know what firmware version was on the board before I reflashed it. I can specify a version that I know works, though that may be a newer version than really required.

Reply:
I think at least doing that would be good.  Saying something about older version might be ok, but good know version is X.Y.Z.